inc/saiqueue.h

```diff
@@ -96,6 +96,28 @@ typedef enum _sai_queue_pfc_continuous_deadlock_state_t
 
 } sai_queue_pfc_continuous_deadlock_state_t;
 
+/**
+ * @brief Enum defining queue actions in case of packet discard.
+ */
+typedef enum _sai_queue_packet_discard_action_t
+{
```

Comment:
Let's discuss the terminology. Have the dillema about discard vs tail-drop vs admission to buffer